src/osp/Resource/PrototypePart.h

```diff
@@ -87,14 +95,8 @@ struct PrototypeObject
     std::variant<DrawableData, ColliderData> m_objectData;
 
     // Put more OSP-specific data in here
-};
-
-using config_node_t = std::variant<double, int, std::string>;
 
-struct PrototypeMachine
-{
-    std::string m_type;
-    std::map<std::string, config_node_t> m_config;
+    std::vector<unsigned> m_machineIndices;
```

Comment:
This is the only functional change to this file; not sure why the other stuff got rearranged (probably during the cherry pick merge). I'll revert those changes before merging as they're just adding noise to an already large commit